src/sentry/scim/endpoints/members.py

```diff
@@ -390,8 +406,8 @@ def post(self, request: Request, organization) -> Response:
             ):
                 # we include conflict logic in the serializer, check to see if that was
                 # our error and if so, return a 409 so the scim IDP knows how to handle
-                raise ConflictError(detail=SCIM_409_USER_EXISTS)
-            return Response(serializer.errors, status=400)
+                raise SCIMApiError(detail=SCIM_409_USER_EXISTS, status_code=409)
```

Comment:
feeling mixed about this, we're trading off automatically setting the status code for automatically setting the schema.

Reply:
this seems cleaner to me!